.classpath

```diff
@@ -4,7 +4,9 @@
 	<classpathentry kind="lib" path="build/dist/lib/coverage.jar"/>
 	<classpathentry kind="lib" path="build/dist/lib/resources.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-	<classpathentry kind="con" path="org.apache.ivyde.eclipse.cpcontainer.IVYDE_CONTAINER/?project=zm-selenium&amp;ivyXmlPath=ivy.xml&amp;confs=*"/>
 	<classpathentry kind="lib" path="build/dist/lib/ZimbraSelenium.jar"/>
+	<classpathentry kind="lib" path="/Users/kumara/work/eclipse-workspace/zm-mailbox/native/build/zm-native-8.8.0.1522920233.jar"/>
```

Comment:
This probably by mistake came and not required to commit.

Reply:
Changed back to original